app/views/_toolbar.html.erb

```diff
@@ -3,65 +3,65 @@
     <% if can? :manage, :all %>
       <li class="dropdown">
         <%= link_to sufia.admin_stats_path, role: 'button', data: { toggle: 'dropdown' }, aria: { haspopup: true, expanded: false } do %>
-          <span class="fa fa-gear"></span> Admin <span class="caret"></span>
+          <span class="fa fa-gear"></span> <%= t("sufia.toolbar.admin.menu") %> <span class="caret"></span>
         <% end %>
         <ul class="dropdown-menu">
           <% if respond_to? :admin_stats_path %>
-            <li><%= link_to 'Repository Statistics', sufia.admin_stats_path %></li>
+            <li><%= link_to t("sufia.toolbar.admin.statistics"), sufia.admin_stats_path %></li>
           <% end %>
 
           <% if respond_to? :admin_queues_path %>
```

Comment:
I'm not a fan of that. I wouldn't mind breaking the admin dropdown partial out. Then you'd only have a few lines to duplicate if you choose to add additional items.